setup.py

```diff
@@ -132,7 +132,9 @@
     #
     # For an analysis of "install_requires" vs pip's requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires=["oemof"],  # Optional
+    install_requires=[
+        "oemof"
+    ],  # Optional  # todo: later exchange to oemof.solph>=0.4.0, oemof.network>=0.4.0 oemof.tools>=??
```

Comment:
@SabineHaas I dont know how this works, but I suppose we need to update the oemof packages here?